src/test/scala/uk/gov/hmcts/ccd/corecasedata/scenarios/WorkBasketInputDetails.scala

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.ccd.corecasedata.scenarios
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import uk.gov.hmcts.ccd.util.{CcdTokenGenerator, PerformanceTestsConfig}
+
+import scala.concurrent.duration._
+
+object WorkBasketInputDetails extends PerformanceTestsConfig {
+
+  private val url: String = config.getString("caseDataUrl") + "/" + config.getString("workbasketInputDetails")
```

Comment:
duplicated code. Why don't you reuse use `caseDataUrl(` method provided by `PerformanceTestsConfig`?

Reply:
it's not duplicate. 
string concatenation to build URLs.